PollyTestClient/Configuration.cs

```diff
@@ -1,16 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace PollyTestClient
+﻿namespace PollyTestClient
 {
     public static class Configuration
     {
         /// <summary>
         /// The root Polly Samples web API site address.
         /// </summary>
-        public const string WEB_API_ROOT = "http://localhost:45178/"; // A variant of this is also deployed at http://pollytestapp.azurewebsites.net
+        public const string WEB_API_ROOT = "http://localhost:45178/";
+
+        // A variant of this is also deployed at
+        //public const string WEB_API_ROOT = "http://pollytestapp.azurewebsites.net";
```

Comment:
What about this? Otherwise it reads to me like the C# syntax is part of the sentence and where it's deployed to.

```suggestion
        // A variant of this is also deployed at "http://pollytestapp.azurewebsites.net";
```

This site also looks a bit broken/outdated, even if it is still there are running. Should we still point people at this @joelhulen?

Reply:
I created that site back in 2015 and haven't touched it since. It is configured to fail after x requests to demonstrate applying policies to remote assets. Technically, there's no difference from executing them with a locally running service. While I agree with your suggestion here, @martincostello, I think we can do away with the reference to that website altogether.

Reply:
Cool - let's just remove this reference then @peter-csala.